src/main/java/gregtech/common/covers/CoverConveyor.java

```diff
@@ -459,6 +458,9 @@ public void readFromNBT(NBTTagCompound tagCompound) {
         this.transferRate = tagCompound.getInteger("TransferRate");
         this.conveyorMode = ConveyorMode.values()[tagCompound.getInteger("ConveyorMode")];
         //LEGACY SAVE FORMAT SUPPORT
+        if(tagCompound.hasKey("ManualImportExportMode")) {
+            this.manualImportExportMode = ManualImportExportMode.values()[tagCompound.getInteger("ManualImportExportMode")];
+        }
```

Comment:
This should be moved somewhere down in this method as it does not belong to LEGACY SAVE FORMAT SUPPORT.